src/analyze/tableau/outlier_detection/README.md

```diff
@@ -0,0 +1,26 @@
+Program outlier_detection_ilf.py process Caliper JSON format data collected from Problem Roulette (PR). Refer to https://github.com/tl-its-umich-edu/caliper-event-analysis/tree/master/src/extract/ for scripts that extract data from Problem Roulette's database.
+
+The program should be excecuted from command line. You will need Python version 3.6+ and packages [pandas, math, sklearn, argparse, json] to run the program.
+
+The program use Isolation Forest algorithm to detect rare ones among all PR problem-dealing events and consider them "outliers". Dimensions used in the to detect outliers include:
```

Comment:
I think a link needs to be provided from where the reading of the Isolation Forest algorithm, what source you have used.